graftm/create.py

```diff
@@ -830,9 +821,23 @@ def main(self, **kwargs):
         temp_output = tempdir.TempDir()
         graftM_graft_test_dir_name = temp_output.name
         temp_output.dissolve()
-        cmd = "graftM graft --forward %s --graftm_package %s --output_directory %s" %(
-            sequences, output_gpkg_path, graftM_graft_test_dir_name)
-        extern.run(cmd)
+        # Take a subset of sequences for testing
+        with tempfile.NamedTemporaryFile(suffix=".fa") as tf:
+            count = 0
+            for name, seq, _ in SequenceIO().each(open(sequences)):
+                count += 1
+                if count < 10:
+                    tf.write(">")
+                    tf.write(s.name)
+                    tf.write("\n")
+                    tf.write(s.seq)
+                    tf.write("\n")
+                else:
+                    break
```

Comment:
Sequence IO has a function for writing sequences: write_fasta_file
Do you mind using that to avoid redundant code?